config.yml

```diff
@@ -310,11 +310,11 @@ scoutsuite:
         - scout --version
 
 serverless:
-  "1.51":
+  "1.57":
```

Comment:
requires to update `.travis.yml` as well (that will fix Travis)

Reply:
:sweat: thanks, repushed